q2cli/builtin/tools.py

```diff
@@ -274,7 +274,7 @@ def _load_metadata(path):
 @tools.command(short_help='View a QIIME 2 Visualization.',
                help="Displays a QIIME 2 Visualization until the command "
                     "exits. To open a QIIME 2 Visualization so it can be "
-                    "used after the command exits, use 'qiime extract'.",
+                    "used after the command exits, use 'qiime tools view --help'.",
```

Comment:
New helptext needs a lil tweak. Take a quick look at the `expected behavior` section of the [original issue](https://github.com/qiime2/q2cli/issues/215). :wink: